projects/core/src/error-handling/effects-error-handler/effects-error-handler.service.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ErrorHandler, Injectable } from '@angular/core';
+import { Action } from '@ngrx/store';
+import { HttpErrorModel } from '@spartacus/core';
+import { HttpErrorResponse } from '@angular/common/http';
+
+@Injectable()
+export class EffectsErrorHandlerService {
+  constructor(protected errorHandler: ErrorHandler) {}
+
+  handleError(action: any): void {
+    const error = this.getError(action);
+    const isNotHttpError =
+      !(error instanceof HttpErrorModel) &&
+      !(error instanceof HttpErrorResponse);
+
+    if (isNotHttpError) {
+      this.errorHandler.handleError(error);
+    }
+  }
+
+  filterActions(_action: Action): boolean {
+    return false;
+  }
```

Comment:
What about moving the default logic here, as well? 
Now this default method looks a bit emtpy...

Reply:
It does look empty, but also I don't think it's a bad thing. Anyways, I've added the default error action types and left a comment.